types/p-lazy/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for p-lazy 1.0
+// Project: https://github.com/sindresorhus/p-lazy#readme
+// Definitions by: BendingBender <https://github.com/BendingBender>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = PLazy;
+
+declare class PLazy<T> extends Promise<T> {
+    static from<T>(fn: () => T | PromiseLike<T>): PLazy<T>;
```

Comment:
Yeah, I know. I hate reading this kind of typings myself. Anyway, thank you for your review!